framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -54,8 +85,9 @@ export const CreatableSelectDropdown = ({
   onOptionChange,
   width = '100%',
   variant = 'outline',
+  currentOption = null,
```

Comment:
Not sure if we have a naming convention here.
Like `initialSelection` or something? do you know @SebCodesTheWeb ?

Reply:
This does not work then. When you take property and use it as default value for `useState`, if the property changes the state value won't be updated.

`const [state, setState ] = useState(defaultValue, ...otherArgs)`

---
```
const { prop1 } = props
const [ state, setState ] = useState(prop1, ...otherArgs) // << prop1 is default state value

setState(123) // << state updated, if you change prop1 from outside, nothing will change here
```

Reply:
You're right 👍